_integration-test/run.sh

```diff
@@ -120,6 +120,7 @@ done
 echo "${_endgroup}"
 
 echo "${_group}Ensure cleanup crons are working ..."
+$dc ps | grep cleanup  # to debug https://github.com/getsentry/self-hosted/issues/1171
```

Comment:
Why not try to see exited containers too while we're at it?
```suggestion
$dc ps -a | grep cleanup  # to debug https://github.com/getsentry/self-hosted/issues/1171
```

Reply:
Sounds good, we need to rerun CI anyway. ;)